ReactWindows/ReactNative/UIManager/PropertySetter.cs

```diff
@@ -114,8 +115,15 @@ protected object ExtractProperty(ReactStylesDiffMap props)
                 return defaultFunc(_attribute);
             }
 
-            return props.GetProperty(Name)?
-                .ToObject(PropertyType);
+            var prop = props.GetProperty(Name);
+            if (prop != null && prop.Type == JTokenType.Array)
```

Comment:
This is going to break all array-based deserialization.  I imagine there's a better spot to fix this.